### ISSUE TITLE
create logic apps project fails with error Unexpected token E in JSON at position 0 

### ISSUE THREAD

**Issue by jomorri** [Azure/logicapps#209]
Does this occur consistently? Yes
Repro steps:
1. Installed Azure Logic Apps Preview
2. Click on the button to create new project

Action: azureLogicAppsV2.createNewProject
Error type: Error
Error Message: Failed to get latest templates: Unexpected token E in JSON at position 0

Version: 0.0.5
OS: win32
OS Release: 10.0.19042
Product: Visual Studio Code
Product Version: 1.51.0
Language: en

<details>
<summary>Call Stack</summary>

```
CentralTemplateProvider.<anonymous> extension.bundle.js:164:430605
fulfilled extension.bundle.js:164:427159
processTicksAndRejections task_queues.js:94:5
```

Here is what I seen in my output window,
6:23:58 PM: Failed to get latest templates: write EPROTO 740473480:error:10000438:SSL routines:OPENSSL_internal:TLSV1_ALERT_INTERNAL_ERROR:../../third_party/boringssl/src/ssl/tls_record.cc:592:SSL alert number 80

6:23:59 PM: Failed to get backup templates: Unexpected token E in JSON at position 0
6:23:59 PM: Error: Failed to get latest templates: write EPROTO 740473480:error:10000438:SSL routines:OPENSSL_internal:TLSV1_ALERT_INTERNAL_ERROR:../../third_party/boringssl/src/ssl/tls_record.cc:592:SSL alert number 80
</details>

**Comment by tonytang-microsoft-com** [comment]
Thanks @jomorri  We are preparing a fix for this issue and we are planning to release a new version ASAP.

**Comment by vikash-anand** [comment]
@jomorri We have released a new version of the extension - v0.0.6 today which should fix this issue. Please update your extension and let us know if the issue exists.
